Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -243,6 +243,17 @@ theorem SameCycle.exists_pow_eq'' [Finite α] (h : SameCycle f x y) :
     · exact ⟨i.succ, i.zero_lt_succ, hi.le, by rfl⟩
 #align equiv.perm.same_cycle.exists_pow_eq'' Equiv.Perm.SameCycle.exists_pow_eq''
 
+instance (f : Perm α) [DecidableRel (SameCycle f⁻¹)] :
+    DecidableRel (SameCycle f) := fun x y =>
```

Comment:
This strikes me as a bad instance, since it will try to find `DecidableRel (SameCycle f)` from `DecidableRel (SameCycle f⁻¹)` from `DecidableRel (SameCycle f⁻¹⁻¹)` from ...

Reply:
I agree with your opinion but this isn't the change in my PR.
For the present, I remove this instance.

Reply:
Ah, I hadn't noticed that, sorry